src/Symfony/Component/Lock/Store/PostgreSqlStore.php

```diff
@@ -108,6 +108,8 @@ public function save(Key $key)
             return;
         }
 
+        $this->getInternalStore()->delete($key);
```

Comment:
shouldn't it be in a try catch block?

Reply:
I added a try/finally block around the code to fetch the advisory lock